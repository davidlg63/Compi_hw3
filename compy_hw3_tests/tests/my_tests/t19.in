void main(){
    {
    auto t = 5 * 5b;
    auto t1 = 5b * 5b;
    auto t2 = 5b * 5;
    auto t3 = 5 / 5b;
    auto t4 = 5b / 5b;
    auto t5 = 5b / 5;
    auto t6 = t5 > t4;
    auto t7 = t4 > t4;
    auto t8 = t4 > t3;
    auto t9 = t2 > t3;
    auto t10 = t2 > t4;
    auto t11 = t10 or t9 and t8;
    }
    {
    auto t = 5 + 5b;
    auto t1 = 5b + 5b;
    auto t2 = 5b + 5;
    auto t3 = 5 - 5b;
    auto t4 = 5b - 5b;
    auto t5 = 5b - 5;
    auto t6 = t5 > t4;
    auto t7 = t4 > t4;
    auto t8 = t4 > t3;
    auto t9 = t2 > t3;
    auto t10 = t2 > t4;
    auto t11 = t10 or t9 and t8;
    }
    {
    auto t = 5 + 5b;
    auto t1 = 5b + 5b;
    auto t2 = 5b + 5;
    auto t3 = 5 - 5b;
    auto t4 = 5b - 5b;
    auto t5 = 5b - 5;
    auto t6 = t5 > t4;
    auto t7 = t4 > t4;
    auto t8 = t4 > t3;
    auto t9 = t2 > t3;
    auto t10 = t2 > t4;
    auto t11 = t10 or t9 and t8;
    }
}