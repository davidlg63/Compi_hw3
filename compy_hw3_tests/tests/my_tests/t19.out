---end scope---
t INT 0
t1 BYTE 1
t2 INT 2
t3 INT 3
t4 BYTE 4
t5 INT 5
t6 BOOL 6
t7 BOOL 7
t8 BOOL 8
t9 BOOL 9
t10 BOOL 10
t11 BOOL 11
---end scope---
t INT 0
t1 BYTE 1
t2 INT 2
t3 INT 3
t4 BYTE 4
t5 INT 5
t6 BOOL 6
t7 BOOL 7
t8 BOOL 8
t9 BOOL 9
t10 BOOL 10
t11 BOOL 11
---end scope---
t INT 0
t1 BYTE 1
t2 INT 2
t3 INT 3
t4 BYTE 4
t5 INT 5
t6 BOOL 6
t7 BOOL 7
t8 BOOL 8
t9 BOOL 9
t10 BOOL 10
t11 BOOL 11
---end scope---
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
main ()->VOID 0
